app/ViewModels/TransactionViewModel.php

```diff
@@ -93,12 +93,12 @@ public function nonce(): int
 
     public function fee(): float
     {
-        return $this->transaction->gas_price->toFloat(); // TODO: https://app.clickup.com/t/86dv41828
+        return $this->transaction->fee()->toFloat();
```

Comment:
this may also be quite slow if it has to do an additional query for every transaction to fetch the corresponding receipt to determine the fee 🤔

Reply:
Yeh 🤔 i might initially get it to pre-load receipts with transactions which will be faster than doing it per transaction (e.g. on the tx page)